oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsDisks.java

```diff
@@ -215,7 +215,7 @@ public static boolean updateDiskStats(HWDiskStore diskStore) {
             diskStore.setWrites(MapUtil.getOrDefault(writeMap, index, 0L));
             diskStore.setWriteBytes(MapUtil.getOrDefault(writeByteMap, index, 0L));
             diskStore.setCurrentQueueLength(MapUtil.getOrDefault(queueLengthMap, index, 0L));
-            diskStore.setTransferTime(MapUtil.getOrDefault(xferTimeMap, index, 0L));
+            diskStore.setTransferTime(MapUtil.getOrDefault(timeStampMap, index, 0L)-MapUtil.getOrDefault(xferTimeMap, index, 0L));
```

Comment:
Very minor performance tweak. If you do the next line (set timestamp) first, then you can just use `diskStore.getTimeStamp()` here.  Also swap lines 249 and 250 for the same reason.